torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10231,8 +10248,11 @@ def ref_pairwise_distance(input1, input2):
                torch.as_strided(x, size, stride, storage_offset=storage_offset),
            dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
            supports_out=False,
+           supports_forward_ad=True,
            sample_inputs_func=sample_inputs_as_strided,
            skips=(
+               # RuntimeError: The size of tensor a (3) must match the size of tensor b (2) at non-singleton dimension
```

Comment:
We should have an inplace formula then that does the right thing? (I guess doing inplace as strided instead of copy_)